app/templates/components/goodcity/offers-search-overlay.hbs

```diff
@@ -6,21 +6,24 @@
           <i {{action "closeOverlay"}} class="back_icon" aria-hidden="true">{{fa-icon 'times' size='lg'}}</i>
         </div>
         <div class="input-holder search-overlay">
-          {{focus-textfield
-            name="searchText"
-            id=uuid
-            placeholder=(t "search_min")
-            value=searchText }}
-
-          {{#if searchText}}
-            <i class="pinned-right" {{action 'clearSearch'}}>{{fa-icon 'times-circle' size='lg'}}</i>
-          {{/if}}
-        </div>
-        {{#if isMobileApp}}
-          <div class="scan-button scanner">
-            {{scan-barcode-button onScanComplete=(action "setScannedSearchText")}}
+          <div class="row">
+            <div class="small-10 columns">
```

Comment:
I think you can keep 12 columns for large screen as we don't have scanner on the large screens. i.e. on web browsers or computers.

Reply:
Sure

Reply:
Also specify columns for medium screen as well.

Reply:
ok